smart-home-provider/frontend/login.html

```diff
@@ -200,17 +200,19 @@
         <img src='images/assistant-small.png' />
 
         <form action='/login' method='post' id='loginform'>
-            <paper-input label="Username" value="rick" name="username">
+            <paper-input label="Username" value="rick" name="paper_username">
                 <iron-icon icon="social:person" prefix></iron-icon>
             </paper-input>
-            <paper-input label="Password" value="oldman" type="password" name="password">
+            <paper-input label="Password" value="oldman" type="password" name="paper_password">
                 <iron-icon icon="communication:vpn-key" prefix></iron-icon>
             </paper-input>
 
             <input type="hidden" name="redirect">
             <input type="hidden" name="client_id">
             <input type="hidden" name="redirect_uri">
             <input type="hidden" name="state" />
+            <input type="hidden" name="username">
+            <input type="hidden" name="password" />
 
             <paper-button raised onclick="formSubmit()">Login</paper-button>
```

Comment:
I know this is a nitpick on myself, but I should remove the inline JS handler and put it instead in a `<script>` block.